app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaPresenter.java

```diff
@@ -68,6 +70,23 @@ public UploadMediaPresenter(UploadRepository uploadRepository,
         this.ioScheduler = ioScheduler;
         this.mainThreadScheduler = mainThreadScheduler;
         compositeDisposable = new CompositeDisposable();
+
+        countryNamesAndCodes = getCountryNamesAndCodes();
```

Comment:
How about initializing this variable only when it is called? (lazy loading)

Reply:
Ok, I rewrote it, that now `countryNamesAndCodes`  will be `null` in beggining, but when it is needed it will be first checked if it is `null` and if yes then it will be filled by calling `getCountryNamesAndCodes()`